keyboards/e6_rgb/e6_rgb.h

```diff
@@ -0,0 +1,88 @@
+ /**
+  * e6_rgb.h
+  *
+  */
+
+#ifndef E6_RGB_H__
```

Comment:
If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`?

It's simpler, and less prone to user error, so if you could make this switch?

```suggestion
#pragma once
```